src/llvm/llvm_installer.ts

```diff
@@ -70,16 +70,16 @@ function nonInteractiveScript(script: string) {
   // make the scirpt non-interactive and fix broken packages
   return script.replace(
     /add-apt-repository "\${REPO_NAME}"/g,
-    // eslint-disable-next-line no-template-curly-in-string
-    "add-apt-repository -y \"${REPO_NAME}\"",
+    `add-apt-repository -y -n "\${REPO_NAME}"
```

Comment:
The downloaded llvm.sh script contains `add-apt-repository -y "${REPO_NAME}"` so I don't think this replacement will be done.
The script also has an update afterwards which could be patch instead of adding the apt-get update. 

See:
https://github.com/opencollab/llvm-jenkins.debian.net/blob/master/llvm.sh#L173-L174

Reply:
Looks like they added the flag recently. The code needs to be updated. Thanks for catching this.
https://github.com/opencollab/llvm-jenkins.debian.net/commit/33f9c64888a7cb5a68c516542917a5328844c03f